.github/actions/isPullRequestMergeable/index.js

```diff
@@ -37,8 +37,15 @@ const run = function () {
                     return;
                 }
 
+                if (_.isNull(data.mergeable_state)) {
+                    console.log('Pull request mergeable_state is not yet resolved...');
+                    retryCount++;
+                    return;
+                }
+
                 mergeabilityResolved = true;
-                isMergeable = data.mergeable;
+                console.log(`Merge information for #${pullRequestNumber} - mergeable: ${data.mergeable}, mergeable_state ${data.mergeable_state}`);
+                isMergeable = data.mergeable && data.mergeable_state !== 'BLOCKED';
```

Comment:
It looks like in the images the value is lowercase, should we uppercase the `data.mergeable_state` value to ensure the comparison is exact?

Reply:
Updated with @roryabraham's suggestion which should handle this